### PR TITLE
Donation Cancellation Confirmation Mail

### PR DESCRIPTION
**Pull request by KaiNissen**
This PR is part of [phab:T123046](https://phabricator.wikimedia.org/T123046).

### CODE REVIEW

File: tests/Integration/UseCases/CancelDonation/CancelDonationUseCaseTest.php [hunk 1]
```diff
@@ -140,7 +140,8 @@ private function newUseCaseWithMailerMock( Donation $donation ): CancelDonationU
 				$this->equalTo( new MailAddress( $donation->getPersonalInfo()->getEmailAddress() ) ),
 				$this->callback( function( $value ) {
 					$this->assertInternalType( 'array', $value );
-					// TODO: assert parameters
+					$this->assertSame( $value['salutation'], 'Sehr geehrte Damen und Herren,' );
+					$this->assertSame( $value['donationId'], 1 );
```
Comment (JeroenDeDauw) [inline tests/Integration/UseCases/CancelDonation/CancelDonationUseCaseTest.php]: Expected first, then actual

**Comment by JeroenDeDauw** [comment]
+1, though someone should double check the template does not say Kai is the emperor of the world in German. We'd not want that to become known to the public

File: tests/Integration/UseCases/CancelDonation/CancelDonationUseCaseTest.php (hunk 1 above)
Comment (KaiNissen) [inline tests/Integration/UseCases/CancelDonation/CancelDonationUseCaseTest.php]: Oops!

**Comment by KaiNissen** [comment]
177476-dott_screenshot

**Pull request merged**